docs/user-guide/01-blueprint-reference.md

```diff
@@ -13,9 +13,9 @@ version = "0.0.1"
 ```
 
 Where:
-- The `name` attribute is a string that contains the name of the blueprint. It can contain spaces, but they will be converted to `-` when it is import into `osbuild-composer`. It should be short and descriptive.
+- The `name` attribute is a string that contains the name of the blueprint. It can contain spaces, but they will be converted to `-` when it is imported into `osbuild-composer`. It should be short and descriptive.
 - The `description` attribute is a string that can be a longer description of the blueprint and is only used for display purposes.
-- The `version` attribute is a string that contains a semver compatible version number. If a new blueprint is uploaded with the same version the server will automatically bump the PATCH level of the version. If the version doesn't match it will be used as is. For example, uploading a blueprint with version set to 0.1.0 when the existing blueprint version is 0.0.1 will result in the new blueprint being stored as version 0.1.0.
+- The `version` attribute is a string that contains a server compatible version number. If a new blueprint is uploaded with the same version the server will automatically bump the PATCH level of the version. If the version doesn't match it will be used as is. For example, uploading a blueprint with version set to 0.1.0 when the existing blueprint version is 0.0.1 will result in the new blueprint being stored as version 0.1.0.
```

Comment:
It really should be "semver" as in "semantic versioning", not server. However, let's not use the abbreviation but instead "semantic versioning" for more clarity.

Reply:
Thank you for clarifying this. I will fix it.